conference/react-js/src/containers/Conference-cards/Past/index.jsx

```diff
@@ -0,0 +1,65 @@
+import React, { PureComponent } from 'react';
+import axios from 'axios';
+import baseUrl from '../../../constants/backend-url';
+
+class Past extends PureComponent {
+  constructor(props) {
+    super(props);
+    this.state = {
+      data: null,
+    };
+  }
+  componentWillMount() {
+    axios.get(`${baseUrl}/api/conference/past`)
+      .then((response) => {
+        const data = response.data;
+        this.setState({ data });
+      });
+  }
+  render() {
+    if (!this.state.data) {
+      return <div />;
+    }
+    const data = this.state.data;
+    const dataArray = Object.values(data);
+    return (
+      <div className="tabs-container">
+        {dataArray.map(element => (
+          <div className="tabs-container conference-card" key={element.id}>
```

Comment:
I recommend moving a card to separate functional component. It will be easier to read.

Reply:
```
{dataArray.map(element => <Card {...element}/>)}
``` will be here. 
And all properties of an element will be props of a Card component.